plugins/SolidView/SolidView.py

```diff
@@ -36,11 +35,13 @@ class SolidView(View):
     """Standard view for mesh models."""
 
     _show_xray_warning_preference = "view/show_xray_warning"
+    _show_overhang_preference = "view/show_overhang"
+    _paint_active_preference = "view/paint_active"
```

Comment:
Seems to be unused, is it an aborted attempt ?

Reply:
Yeah, well, close enough to that -- it was sort of like the current attempt kept evolving to the point where I didn't need `SolidView` _at all_ anymore (which I think would eventually happen, maybe just not in this ticket already).